.eslintignore

```diff
@@ -1,2 +1,3 @@
 # output folder
 dist/
+rollup.config.js
```

Comment:
"CI Standard Checks" complains about a JS file unless it is here in igrnore file.